devex/cross-nuget/README.md

```diff
@@ -0,0 +1,136 @@
+   ```powershell
+   New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Services\sgx_lc_msr\Parameters" [...]
+   ```
+4. Execute the script as follows:
+   ```powershell
+   Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process
+   .\install-windows-prereqs.ps1 -InstallPath C:/oe_prereqs -LaunchConfiguration SGX1FLC-NoIntelDrivers -DCAPClientType None
+   ```
+5. When prompted, reboot the system.
+
+### Ubuntu
+
+1. Run `setup-host.sh`, accepting the LXD initialization defaults.
+2. Log out and log back in.
+3. Run `setup-containers.sh` to create and start the containers used for
+   building; unlike Docker containers, these remain running in the background
+   and have persistent storage; think of them as virtual machines instead of
+   containers.
+4. Clone the SDK:
+   ```bash
+   git clone -b v0.11.0 --recursive --depth=1 https://github.com/openenclave/openenclave sdk
+   ```
+5. Open `scripts/ansible/oe-contributors-setup.yml` and comment out the step
+   that installs the Intel SGX driver because drivers cannot be installed inside
+   containers.
+6. Install the prerequisites in each container:
+   ```bash
+   # Enter the Xenial container.
+   lxc exec oepkgxenial su ubuntu
+
+   cd /host/path/to/sdk
+   sudo scripts/ansible/install-ansible.sh
+   ansible-playbook scripts/ansible/oe-contributors-setup-cross-arm.yml
+
+   sudo apt install python3-pip p7zip-full -y
+   pip3 install pycryptodome
+
+   # Quit the Xenial container.
+   exit
+
+   # Enter the Bionic container.
+   lxc exec oepkgbionic su ubuntu
+
+   cd /host/path/to/sdk
+   sudo scripts/ansible/install-ansible.sh
+   ansible-playbook scripts/ansible/oe-contributors-setup-cross-arm.yml
+   sudo apt install python3-pyelftools p7zip-full -y
+
+   # Quit the Bionic container.
+   exit
+   ```
+
+## Building
+
+### Windows
+
+To build the SDK in all its configurations:
+
+1. Launch the x64 Native Tools Command Prompt for VS 2017 from the Start menu.
+2. Run:
+   ```cmd
+   cd %SYSTEMDRIVE%\path\to\this\repository\windows
+   powershell -c "Set-ExecutionPolicy Bypass -Scope Process; .\build.ps1"
+   powershell -c "Set-ExecutionPolicy Bypass -Scope Process; .\pack.ps1"
+   ```
+
+This creates a `Pack` directory with the build outputs.
+
+### Ubuntu
+To build the SDK in all its configurations, run:
+
+```bash
+cd /path/to/this/repository/linux
+./driver.sh
+./pack.sh
+```
+
+This creates a `pack` directory with the build outputs.
+
+## Packaging
+
+Fetch the resulting `pack` directories from both environments and merge their
+contents. This can be achieved by copying the contents of one folder into the
+other; there are no files that overlap.
+
+Then, copy the contents of the `devex/vsextension/msbuild/` directory into the
+`pack/build/native` directory. Additionally, copy the contents of the
+`devex/cross-nuget/extras` directory into the `pack` directory. Update the
+version information insde `open-enclave-cross.nuspec` if necessary.
```

Comment:
When is it necessary to update the version?

Reply:
Added a section to the `README` with details on versioning. Do you think it's clear now?